corehq/apps/reports/commtrack/data_sources.py

```diff
@@ -243,7 +243,7 @@ def filter_by_program(self, stock_states):
             )
         )
 
-    def get_data(self, slugs=None):
+    def get_data(self):
```

Comment:
@dannyroberts @snopoke @gcapalbo 
It seems slugs is still used here:
https://github.com/dimagi/commcare-hq/pull/9272/files#diff-7300848b9db6b7a41ec64acb711b91bfR275